collector.go

```diff
@@ -11,122 +10,135 @@ import (
 )
 
 var (
-	templates = map[string]*cwCollectorTemplate{}
+	metrics = []string{
+		"ActiveTransactions",
+		"AuroraBinlogReplicaLag",
+		"AuroraReplicaLag",
+		"AuroraReplicaLagMaximum",
+		"AuroraReplicaLagMinimum",
+		"BinLogDiskUsage",
+		"BlockedTransactions",
+		"BufferCacheHitRatio",
+		"BurstBalance",
+		"CommitLatency",
+		"CommitThroughput",
+		"CPUCreditBalance",
+		"CPUCreditUsage",
+		"CPUUtilization",
+		"DatabaseConnections",
+		"DDLLatency",
+		"DDLThroughput",
+		"Deadlocks",
+		"DeleteLatency",
+		"DeleteThroughput",
+		"DiskQueueDepth",
+		"DMLLatency",
+		"DMLThroughput",
+		"EngineUptime",
+		"FreeableMemory",
+		"FreeLocalStorage",
+		"FreeStorageSpace",
+		"InsertLatency",
+		"InsertThroughput",
+		"LoginFailures",
+		"NetworkReceiveThroughput",
+		"NetworkThroughput",
+		"NetworkTransmitThroughput",
+		"Queries",
+		"ReadIOPS",
+		"ReadLatency",
+		"ReadThroughput",
+		"ResultSetCacheHitRatio",
+		"SelectLatency",
+		"SelectThroughput",
+		"SwapUsage",
+		"UpdateLatency",
+		"UpdateThroughput",
+		"VolumeBytesUsed",
+		"VolumeReadIOPs",
+		"VolumeWriteIOPs",
+		"WriteIOPS",
+		"WriteLatency",
+		"WriteThroughput",
+	}
```

Comment:
For PMM-1639 that should be replaced with mapping of CloudWatch metrics names to proper Prometheus metric names (native where possible) and types.